gradle.properties

```diff
@@ -18,4 +18,4 @@ POM_DEVELOPER_ID2=uKL
 POM_DEVELOPER_NAME2=Paweł Urban
 
 android.useAndroidX=true
-android.enableJetifier=true
+android.enableJetifier=false
```

Comment:
I would keep it as `true` as long as Dagger2 does not support AndroidX :-)

Reply:
Hmmm... That's a good call — our shadowed version of Dagger2 may have not been properly checked. Isn't AndroidX supported in Dagger2 since `2.20`?

Reply:
Well, hard to say. The latest version of `dagger2` is as I see `2.22.1` so I would update it to that version and just check it, but there is no official info about full androidx support: https://github.com/google/dagger/issues/1188. 
Ron Shapiro in [#188](https://github.com/google/dagger/issues/1188) suggest using jetifier by default.

Reply:
I remember that few month ago I had issues with `2.16` version and setting `enableJetifier ` to `true` fixed the issue. I do not have experience with later versions as nowadays I am working on projects with `koin`.

Reply:
I have just checked with a fresh project and `1.9.2-SNAPSHOT` version:
```
./gradlew -Pandroid.enableJetifier=false canIDropJetifier

> Task :app:canIDropJetifier
========================================
Project app
========================================

No dependencies on old artifacts! Safe to drop Jetifier.
```
It compiled and run successfully. I suppose it is because the library does not use any support package classes and Dagger does not need to generate any code for them. No usage of those classes in generated code => no need for Jettifier.

Reply:
Anyway — thank you for double checking me 🥇